tests/integration/conftest.py

```diff
@@ -26,6 +27,19 @@ def app_name(metadata):
     yield metadata["name"]
 
 
+@fixture(scope="module")
+def run_action(ops_test: OpsTest):
```

Comment:
Have you considered adding the return type here?

Reply:
Type added